packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import DataGrid from './DataGrid';
+import rowShape from './proptypes/rowShape';
+import { columnShape } from './proptypes/columnShape';
+import styles from './FlowsheetDataGrid.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * String that identifies the element (or elements) that labels the grid.
+   */
+  ariaLabelledBy: PropTypes.string,
+
+  /**
+   * String that labels the grid for accessibility. If ariaLabelledBy is specified, ariaLabel will not be used.
+   */
+  ariaLabel: PropTypes.string,
+
+  /**
+   * String that will be used to identify the Grid. If multiple grids are on the same page, each grid should have
+   * a unique id.
+   */
+  id: PropTypes.string.isRequired,
+
+  /**
+   * Data for content in the body of the Grid. Rows will be rendered in the order given. The first cell in each row will be the row header.
+   */
+  rows: PropTypes.arrayOf(rowShape),
+
+  /**
+   * The column containing row headers. This column will be pinned to the leftmost side of the grid.
+   */
+  rowHeaderColumn: columnShape,
```

Comment:
Instead of having a separate prop, can we just enforce that the first column of the columns array is the pinned column?  Then consumers just need to create a single array.